CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ This project primarily welcomes contributions of two types:
 - Pull Requests for pre-approved features, which are based on BC Ideas (http://aka.ms/bcideas) that have been picked as candidates for code contribution projects.
 - Pull Requests with fixes and improvements addressing pre-approved issues, which are non-customer reported.
 
-In either case, **an approved GitHub issue in state "approved", must exist, before a Pull Request can be created**. Once a Pull Request is created it **must** be linked to a GitHub issue. 
+In either case, **an approved GitHub issue in state "approved", must exist, before a Pull Request can be created**. Once a Pull Request is created it **must** be linked to a GitHub issue.
```

Comment:
We can probably add example how to link the issue ("Resolves #xxx")?

Reply:
Added some clarifications. Thanks!